presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1943,11 +1943,6 @@ public void testGroupingWithFortyArguments()
                 groupingSet2);
 
         assertQuery(query, "VALUES (0), (822283861886), (995358664191)");
-
-        // GROUPING + implicit coercions (issue #8738)
-        assertQuery(
-                "SELECT GROUPING(k), SUM(v) + 1.0 FROM (VALUES (1, 1)) AS t(k,v) GROUP BY k",
-                "VALUES (0, 2.0)");
```

Comment:
i ACK this didn't really belong to `testGroupingWithFortyArguments` test, but maybe the whole `testGroupingWithFortyArguments` test belong to the new `TestGrouping` class? Looks like it exercises engine only.

Reply:
It does. I'll start moving things out later.

Reply:
If you don't plan to move all grouping tests now, I'd abstain from this refactor until this "later". But that's just an opinion, up to you.